src/App.tsx

```diff
@@ -177,14 +178,15 @@ export function App() {
 
                 return (
                   <ButtonsSection>
-                    {filtered.map((sound) => (
+                    {filtered.map(({ item }: any) => (
```

Comment:
you definitely shouldn't need this

Reply:
please type the `filtered` array correctly and don't use any here

Reply:
done